<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TacoTranscribe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>691</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="jpnTextDisplay">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="engTextDisplay">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>691</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="OggFileName">
       <property name="text">
        <string>audiofile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AudioGraph" name="audioGraphWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsTop">
       <item>
        <widget class="QPushButton" name="replayBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Play assigned audio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NextBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save, Go to next line</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reassignAudioBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Reassign audio file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="audioMissingBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Mark audio missing</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="newEngText"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ButtonsBottomTop">
       <item>
        <widget class="QPushButton" name="playNextBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Play next audio file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shiftNegBut">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Shift audio file assignment by -1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shiftPosBut">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Shift audio file assignment by +1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="audioUsuitedSkip">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Audio not suitable</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Audio"/>
    <addaction name="actionLoad_Script"/>
    <addaction name="actionLoad_Savept"/>
    <addaction name="actionSaveScript"/>
    <addaction name="actionSaveSavepoint"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Savept">
   <property name="text">
    <string>Load Savepoint</string>
   </property>
  </action>
  <action name="actionSaveScript">
   <property name="text">
    <string>Save Script</string>
   </property>
  </action>
  <action name="actionSaveSavepoint">
   <property name="text">
    <string>Save Savepoint</string>
   </property>
  </action>
  <action name="actionLoad_Script">
   <property name="text">
    <string>Load Script</string>
   </property>
  </action>
  <action name="actionLoad_Audio">
   <property name="text">
    <string>Load Audio</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AudioGraph</class>
   <extends>QWidget</extends>
   <header>audiograph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
